vault.go

```diff
@@ -212,12 +466,38 @@ func doVaultConfiguration(vc *vault.Client, profileName string, minioInstances [
 	//
 	// Add MinIO role
 	//
-	for _, instance := range minioInstances {
-		if err := doMinioRole(vc, instance, name); err != nil {
+	//for _, instance := range vc.MinioInstances {
```

Comment:
Is there a reason this is commented out?

Reply:
Fat fingers!